sample-action-client/package.json

```diff
@@ -72,10 +72,8 @@
 		"onCommand:contrib.action.open.settings",
 		"onCommand:contrib.action.open.file"
 	],
-	"dependencies": {
-		"@sap-devx/app-studio-toolkit-types": "0.0.7"
-	},
 	"devDependencies": {
+		"@sap-devx/app-studio-toolkit-types": "0.0.11",
```

Comment:
👍

Reply:
I think you need to use the following form while the project is not yet a mono-repo
- `		"@sap-devx/app-studio-toolkit-types": "file:./types",`